torchgeo/datasets/esri2020.py

```diff
@@ -136,3 +137,52 @@ def _download(self) -> None:
     def _extract(self) -> None:
         """Extract the dataset."""
         extract_archive(os.path.join(self.root, self.zipfile))
+
+    def plot(  # type: ignore[override]
+        self,
+        sample: Dict[str, Tensor],
```

Comment:
```suggestion
        sample: Dict[str, Any],
```
`__getitem__` doesn't even return `Dict[str, Tensor]`, the sample often contains non-Tensors like CRS and BoundingBox.

Reply:
Okay, should I change it in all the previously added plot methods like spacenet and so on as well?

Reply:
does it also require a version changed comment?